install_scripts/install_flask.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "intsalling flask prerequisites"
+
+echo "installing python 3.7"
+sudo apt-get install -y python3.7
+
+echo "installing python3-pip"
+sudo apt-get install -y python3-pip
+
+echo "installing flask"
+pip3 install flask
```

Comment:
In Pull Request 18 (https://github.com/knowabli/edison/pull/18) there's a requirements.txt file added that installs flask used in the CI. Can the team make a decision if you want to install software packages piecemeal (as you are doing here in multiple files) or all at once using a requirements.txt file.

The reason why would would want to do it piecemeal (as is done here) is if you have a different server for the database and webapp. However, if you are planning on having it all on a single server, it might be worth having the requirements.txt file.

Reply:
I don't know the amount of the servers that allocated for this project.. its redhat decision no?

Reply:
Its your team's decision how you set up the project. You can set it up for a single server because you have no plan on its growing, and if it ever grows refactor (more agile), or you can say, my expectation is that it will grow into a larger project where we should plan on separating the components now because it will be more difficult later.
For multiple servers in your current environment, you can always ramp up two VMs on vagrant, so it is not dependent on Red Hat at all.

Reply:
In any case, my point is that there should be a single unified method of installation, rather than one team member using a requirements file and other team members

Reply:
Done